docs/source/azure/authentication.rst

```diff
@@ -50,26 +67,16 @@ You will need to map it to environment variables for Custodian like this:
     AZURE_CLIENT_ID=appId
     AZURE_CLIENT_SECRET=password
 
+Created Service Principal already has Contributor role.
```

Comment:
Is this true for every tenant's subscription?

Reply:
Never mind - it is! https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md